static/css/media-queries.css

```diff
@@ -0,0 +1,28 @@
+/*  We can't use standard breakpoints as the left panel navigation in Jira hijacks some of the screen */
+@media (max-width: 1530px) {
+    .jiraConfiguration__table__cell__settings__ellipsis {
+      right: 0;
+    }
+  }
```

Comment:
Can't say I'm a big fan of putting the media queries in a separate file.  I much prefer placing it in the same file where the original styles are.  Seems like more work when putting them in a different file.

Reply:
Ok, I'll look at moving these styles back out into the other css files in the PR I have open for connect an org pg.